collector/drbd_linux.go

```diff
@@ -67,73 +67,73 @@ func newDrbdStringPairMetric(name string, desc string, valueOkay string) drbdStr
 
 var (
 	drbdNumericalMetrics = map[string]drbdNumericalMetric{
-		"ns": newDrbdNumericalMetric(
-			"network_sent_bytes",
-			"Volume of net data sent to the partner via the network connection.",
+		"ns": newDRBDNumericalMetric(
+			"network_sent_bytes_total",
+			"Volume of net data sent to the partner via the network connection; in bytes.",
```

Comment:
Sorry for nitpicking, but I find the wording a bit odd and redundant. What do you think about

"Total number of bytes sent via the network"?

Same for `received` and similar for `disk_` metrics ("Total number of bytes read from/written to disk")